src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,244 @@
+import Breadcrumbs from "core/components/Breadcrumbs";
+import Block from "core/components/Block";
+import DataGrid, { BaseColumn } from "core/components/DataGrid";
+import ChevronLinkColumn from "core/components/DataGrid/ChevronLinkColumn";
+import DateColumn from "core/components/DataGrid/DateColumn";
+import Page from "core/components/Layout/Page";
+import { PageContent } from "core/components/Layout/PageContent";
+import { createGetServerSideProps } from "core/helpers/page";
+import {
+  DashboardPageDocument,
+  useDashboardPageQuery,
+} from "dashboards/graphql/queries.generated";
+import { useTranslation } from "next-i18next";
+import { BookmarkSquareIcon, BookOpenIcon } from "@heroicons/react/24/outline";
+import DescriptionList, {
+  DescriptionListDisplayMode,
+} from "core/components/DescriptionList";
+import Link from "next/link";
+import { ActivityStatus } from "graphql-types";
+import Badge from "core/components/Badge";
+import { useMemo } from "react";
+import clsx from "clsx";
+import { capitalize } from "lodash";
+
+const ActivityStatusBadge = ({ status }: { status: ActivityStatus }) => {
+  const { t } = useTranslation();
+  let className = useMemo(() => {
+    switch (status) {
+      case ActivityStatus.Error:
+        return "bg-red-100 text-red-500";
+      case ActivityStatus.Pending:
+        return "bg-gray-100 text-gray-600";
+      case ActivityStatus.Running:
+        return "bg-sky-100 text-sky-600";
+      case ActivityStatus.Success:
+        return "bg-emerald-50 text-emerald-500";
+    }
+  }, [status]);
+  return (
+    <Badge className={clsx(className, "flex items-center")}>
+      {t(capitalize(status))}
```

Comment:
That will not work since what will be exported as translatable would be `capitalize(status)`.

You need to have a switch or a map for the labels

Reply:
Got it.

Reply:
I'll try with a switch